docs/source/getting-started/images/gsg_architecture.svg

```diff
@@ -1 +1 @@
x(0.05,0,0,0.05,36.95,0)" xlink:href="#ae"/><use transform="matrix(0.05,0,0,0.05,40.900000000000006,0)" xlink:href="#N"/><use transform="matrix(0.05,0,0,0.05,50.900000000000006,0)" xlink:href="#M"/><use transform="matrix(0.05,0,0,0.05,60.900000000000006,0)" xlink:href="#N"/></g><g id="u"><use transform="matrix(0.05,0,0,0.05,0,0)" xlink:href="#ah"/><use transform="matrix(0.05,0,0,0.05,12.950000000000001,0)" xlink:href="#J"/><use transform="matrix(0.05,0,0,0.05,22.950000000000003,0)" xlink:href="#L"/><use transform="matrix(0.05,0,0,0.05,37.900000000000006,0)" xlink:href="#ai"/><use transform="matrix(0.05,0,0,0.05,47.900000000000006,0)" xlink:href="#J"/><use transform="matrix(0.05,0,0,0.05,57.900000000000006,0)" xlink:href="#Q"/><use transform="matrix(0.05,0,0,0.05,67.9,0)" xlink:href="#M"/><use transform="matrix(0.05,0,0,0.05,77.9,0)" xlink:href="#Q"/><use transform="matrix(0.05,0,0,0.05,87.9,0)" xlink:href="#H"/></g><path d="M143 4C61 4 22-44 18-125c-5-107 100-154 193-111 17 8 29 25 37 43l-32 9c-13-25-37-40-76-40-61 0-88 39-88 99 0 61 29 100 91 101 35 0 62-11 79-27v-45h-74v-28h105v86C228-13 192 4 143 4" id="ak"/><g id="v"><use transform="matrix(0.05,0,0,0.05,0,0)" xlink:href="#ak"/><use transform="matrix(0.05,0,0,0.05,14,0)" xlink:href="#M"/><use transform="matrix(0.05,0,0,0.05,24,0)" xlink:href="#Q"/><use transform="matrix(0.05,0,0,0.05,34,0)" xlink:href="#M"/><use transform="matrix(0.05,0,0,0.05,44,0)" xlink:href="#K"/><use transform="matrix(0.05,0,0,0.05,49.95,0)" xlink:href="#G"/><use transform="matrix(0.05,0,0,0.05,59.95,0)" xlink:href="#H"/><use transform="matrix(0.05,0,0,0.05,64.95,0)" xlink:href="#M"/><use transform="matrix(0.05,0,0,0.05,74.95,0)" xlink:href="#N"/></g><g id="w"><use transform="matrix(0.05,0,0,0.05,0,0)" xlink:href="#I"/><use transform="matrix(0.05,0,0,0.05,5,0)" xlink:href="#K"/><use transform="matrix(0.05,0,0,0.05,10.95,0)" xlink:href="#J"/><use transform="matrix(0.05,0,0,0.05,20.95,0)" xlink:href="#L"/></g><g id="x"><use transform="matrix(0.05,0,0,0.05,0,0)" xlink:href="#A"/><use transform="matrix(0.05,0,0,0.05,12.950000000000001,0)" xlink:href="#B"/><use transform="matrix(0.05,0,0,0.05,24.950000000000003,0)" xlink:href="#C"/><use transform="matrix(0.05,0,0,0.05,39.900000000000006,0)" xlink:href="#D"/></g><g id="y"><use transform="matrix(0.05,0,0,0.05,0,0)" xlink:href="#L"/><use transform="matrix(0.05,0,0,0.05,14.950000000000001,0)" xlink:href="#J"/><use transform="matrix(0.05,0,0,0.05,24.950000000000003,0)" xlink:href="#N"/><use transform="matrix(0.05,0,0,0.05,34.95,0)" xlink:href="#M"/><use transform="matrix(0.05,0,0,0.05,44.95,0)" xlink:href="#F"/></g><g id="z"><use transform="matrix(0.05,0,0,0.05,0,0)" xlink:href="#C"/><use transform="matrix(0.05,0,0,0.05,14.950000000000001,0)" xlink:href="#J"/><use transform="matrix(0.05,0,0,0.05,24.950000000000003,0)" xlink:href="#N"/><use transform="matrix(0.05,0,0,0.05,34.95,0)" xlink:href="#M"/><use transform="matrix(0.05,0,0,0.05,44.95,0)" xlink:href="#F"/></g></defs></g></svg>
```

Comment:
The two images seem inconsistent. In the GSG, Sandbox is the network. In the app architecture guide, the participant is part of the application backend. This is particularly confusing since the app architecture guide goes on to talk about Sandbox.

Reply:
It is a bit messy. See the new diagram and text on the ecosystem overview page. I no longer consider the participant being part of the DAML Network, but in Sandbox it's fully integrated. I guess we could call Sandbox a mock Participant Node and move it into the Application Backend.

Reply:
I think we should at least point this out the two styles of integrations more explicitly here. DAML on SQL and vDAML is what people actually use here and there the participant is on the network side.

Reply:
@cocreature How's this updated version?